pkgs/applications/editors/vim/plugins/vim-plugin-names

```diff
@@ -356,6 +359,7 @@ https://github.com/cocopon/iceberg.vim/,,
 https://github.com/idris-hackers/idris-vim/,,
 https://github.com/edwinb/idris2-vim/,,
 https://github.com/3rd/image.nvim/,HEAD,
+https://github.com/samodostal/image.nvim/,HEAD,
```

Comment:
we already have a `image-nvim`, please consider giving this an alias

Reply:
updated this with the alias `samodostal-image-nvim`